src/main/java/org/ethereum/beacon/discovery/pipeline/handler/OutgoingParcelHandler.java

```diff
@@ -44,8 +44,14 @@ public void handle(Envelope envelope) {
                 envelope.getId()));
 
     if (envelope.get(Field.INCOMING) instanceof NetworkParcel) {
-      outgoingSink.next((NetworkParcel) envelope.get(Field.INCOMING));
-      envelope.remove(Field.INCOMING);
+      NetworkParcel parcel = (NetworkParcel) envelope.get(Field.INCOMING);
+      if (parcel.getPacket().getBytes().size() > IncomingDataPacker.MAX_PACKET_SIZE) {
+        logger.debug(
```

Comment:
Seems like this should be an error:
```suggestion
        logger.error(
```